### PR TITLE
Improve the syntax of the README file

### PR DESCRIPTION
**Pull request by berendt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+It allows for easy configuring of what the test documents look like, what
+kind of data types they include and what the field names are called.
 
 ## Cool, how do I use this? 
 
-#### Prerequisites
-
-Let's assume you have an Elasticsearch cluster running. If not, [set it up locally](http://ohardt.us/es-install) and point your browser to [http://localhost:9200](http://localhost:9200) to see if it's up.
+Let's assume you have an Elasticsearch cluster running. If not,
+[set it up locally](http://ohardt.us/es-install) and point your browser to
```
Comment (oliver006) [inline README.md]: Can you please remove this link (http://ohardt.us/es-install) and the other one down in line 82 ? They are outdated and it'd be better to remove them.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
+[http://localhost:9200](http://localhost:9200) to see if it's up.
 
-Python and [Tornado](https://github.com/tornadoweb/tornado/) and [NumPy](http://www.numpy.org/) are used, run `pip install tornado numpy` to install Tornado and NumPy if you don't have it already.
+Python and [Tornado](https://github.com/tornadoweb/tornado/) and
+[NumPy](http://www.numpy.org/) are used, run `pip install tornado numpy` to
```
Comment (oliver006) [inline README.md]: The numpy dependency was removed in https://github.com/oliver006/elasticsearch-test-data/commit/4d31f6742e62cd992b7fb30612894abd640495f9 - you can remove all references to it from the README.

File: README.md [hunk 3]
```diff
@@ -51,52 +56,70 @@ Without any command line options, it will generate and upload 1000 documents of
...
-- `--batch_size=###` we use bulk upload to send the docs to ES, this option controls how many we send at a time
+  - `--num_of_replicas=0` the number of replicas for the index
+- `--batch_size=###` we use bulk upload to send the docs to ES, this option
+  controls how many we send at a time
 - `--force_init_index=False` if `True` it will delete and re-create the index
-- `--dict_file=filename.dic` if provided the `dict` data type will use words from the dictionary file, format is one word per line. The entire file is loaded at start-up so be careful with (very) large files. You can download wordlists e.g.. from [here](http://ohardt.us/word-lists). 
+- `--dict_file=filename.dic` if provided the `dict` data type will use words
+  from the dictionary file, format is one word per line. The entire file is
+  loaded at start-up so be careful with (very) large files. You can download
+  wordlists e.g. from [here](http://ohardt.us/word-lists).
```
Comment (oliver006) [inline README.md]: Remove the link to http://ohardt.us/word-lists

**Review by oliver006 — COMMENTED** [review]
Looks good, couple of changes and this should be good to go.

**Comment by berendt** [comment]
> Looks good, couple of changes and this should be good to go.

Done.

**Review by oliver006 — APPROVED** [review]
Nice work, thanks for the PR!

**Pull request merged**